src/bake/DevServer/HmrSocket.zig

```diff
@@ -27,7 +27,7 @@ pub fn new(dev: *DevServer, res: anytype) *HmrSocket {
     });
 }
 
-pub fn onOpen(s: *HmrSocket, ws: AnyWebSocket) void {
+pub fn onOpen(s: *HmrSocket, ws: AnyWebSocket) error{}!void {
```

Comment:
i've never seen this syntax

i won't be the only person to say that and wonder why an error union should be able to be empty

can we not do that? or if it's to conform to some interface, can we make it so it detects if it's error-able or not and propagate accordingly?